### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by hefler**
Some UX improvements: 
* Moved crumbs to the bottom, avoiding interface jittering when crumbs change in size.
* Crumbs will scroll laterally instead of having multiple lines.
* Font size will honour interface size (they were way too big).

### CODE REVIEW

**Review by ocoka — COMMENTED** [review]
Hello, and thank's for your interest to this project and your contribution, I will look into the matter of your changes, but for now I can say that UX should be configurable in case of crumbs positionning, I think some users liked crumbs panel to be on top

**Comment by ocoka** [comment]
image
I've tested it on my Atom, some styling issue with positioning

File: lib/indent-codecrumbs-view.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default class IndentCodeCrumbsView {
         {
           this.props.crumbs.map(_ =>
             <li on={{click: () => this.props.crumbsClick(_)}} className={`indent-codecrumbs__el ${this.props.color ? _.scopeClasses : ""}`}>
-              {_.title}
+              <span>{_.title}</span>
```
Comment (ocoka) [inline lib/indent-codecrumbs-view.js]: We are heading to use best practices for frontend developing, like BEM naming in CSS class names (for details see: https://en.bem.info/methodology/quick-start/#contents)

```suggestion
              <span className='indent-codecrumbs__crumb-text'>{_.title}</span>
```

**Comment by hefler** [comment]
> image
> I've tested it on my Atom, some styling issue with positioning

This looks like some issue with the `indent-collector` it's generating empty crumbs.

**Pull request merged**